### PR TITLE
chore(it): cleanup it poms

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: integration-tests/jdbc/pom.xml [hunk 1]
```diff
@@ -31,6 +31,10 @@
     <name>Camel Quarkus :: Integration Tests :: JDBC</name>
     <description>Integration tests for Camel JDBC extension</description>
 
+    <properties>
+        <!-- Skip enforcer plugin as we want to use jboss marshalling for test class -->
+        <enforcer.skip>true</enforcer.skip>
```
Comment (ppalaga) [inline integration-tests/jdbc/pom.xml]: Skipping the enforcer was not necessary in the JDBC itest, IIRC.

File: integration-tests/servlet/pom.xml [hunk 2]
```diff
@@ -31,6 +31,11 @@
     <name>Camel Quarkus :: Integration Tests :: Servlet</name>
     <description>Integration tests for Camel Servlet component</description>
 
+    <properties>
+        <!-- Skip enforcer plugin as we want to use jboss marshalling for test class -->
+        <enforcer.skip>true</enforcer.skip>
```
Comment (ppalaga) [inline integration-tests/servlet/pom.xml]: Same here: it was passing with the enforcer on, IIRC. Are there other reasons for disabling it?

Comment (lburgazzoli) [reply integration-tests/servlet/pom.xml]: I found this comment on other poms so maybe it is not needed at all

**Comment by lburgazzoli** [comment]
@ppalaga ok, I've disabled enforcer only were it make sense

**Pull request merged**